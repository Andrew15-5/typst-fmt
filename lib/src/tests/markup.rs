use super::*;
make_test!(unchanged, " ");
make_test!(one_space, "  ");
make_test!(one_space_b, "   ");
make_test!(two_line_max, "\n\n\n");
make_test!(
    content_block_basic,
    "#[\n_Glaciers_ form an important part \nof the earth's climate system.]"
);
make_test!(
    content_block2,
    "#[
    something
    ]"
);
make_test!(content_block_spaced, "[ 4 ]");
make_test!(content_block_too_spaced, "[  4  ]");
make_test!(content_block_tight, "[4]");
make_test!(content_block_only_space, "[   ]");
make_test!(line_wrapping, "Lorem _ipsum_ dolor sit amet, _consectetur_ adipiscing elit, sed do eiusmod tempor incididunt ut labore.");
make_test!(slash_space, r"#[\ ]");
make_test!(
    text_then_list,
    "We have next things:
- thing 1;
- thing 2;
- thing 3."
);
make_test!(bug_quote_space, r#"don't "text" "text"#);
make_test!(bug_space_around_inline_code, "a#lorem(2)b c d");
