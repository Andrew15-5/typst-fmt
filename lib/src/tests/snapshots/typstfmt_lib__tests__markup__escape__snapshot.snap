---
source: lib/src/tests/markup.rs
description: "INPUT\n===\n\"C\\\\#\"\n===\nC\\#\n===\nFORMATTED\n===\nC\\#"
expression: formatted
---
"C\\#"
